jerry-core/jerry-api.h

```diff
@@ -147,71 +173,83 @@ jerry_value_t jerry_value_to_string (const jerry_value_t);
 jerry_value_t jerry_value_remove_error_flag (const jerry_value_t);
```

Comment:
Perhaps jerry_value_clear_error_flag is a better name here. And we could add a set_error_flag as well.